lib/screens/explore_screens/explore/explore_screen.dart

```diff
@@ -94,11 +100,7 @@ class ExploreScreen extends StatelessWidget {
                     backgroundImage: i.backgroundImage,
                     gradient: i.gradient,
                     onTap: () {
-                      if (i.route.isNotEmpty) {
-                        BlocProvider.of<ExploreBloc>(context).add(const OnExploreCardTapped(Routes.createInvite));
```

Comment:
this was a bug, always navigate to Routes.createInvite instead of the passed invite.